tests/benchmark/utils/shared_info.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <s2n.h>
+#include <benchmark/benchmark.h>
+#include <netdb.h>
+#include <getopt.h>
+#include "tls/s2n_cipher_suites.h"
+#include "tls/s2n_cipher_preferences.h"
+
+extern struct s2n_cipher_suite **all_suites;
+extern unsigned int num_suites;
+extern int DEBUG_PRINT;
+extern const char *host;
+extern const char *port;
+extern const char *pem_dir;
+extern int sockfd, fd_bench;
```

Comment:
Are these really shared? This seems to only be used by the server. Also can fd_bench have a better name? Is it the connection fd (file descriptor)?